src/asm/rgbasm.5

```diff
@@ -875,43 +875,45 @@ and at signs
 However, they must begin with either a letter or an underscore.
 Periods
 .Sq \&.
-are allowed exclusively in labels, as described below.
+are allowed exclusively in label names, as described below.
 A symbol cannot have the same name as a reserved keyword.
-.Ss Label declaration
+.Ss Label definition
 One of the assembler's main tasks is to keep track of addresses for you, so you can work with meaningful names instead of "magic" numbers.
+Labels enable just that.
+A label definition is made of a label name at the beginning of a line,
+.Sy immediately
+followed by one or two colons.
```

Comment:
```suggestion
A label definition is made of a label name at the beginning of a line,
.Sy immediately
followed by one or two colons.
```
A label definition is made of a label name at the beginning of a line, followed by one or two colons (without any whitespace between the label name and the colons).